src/lib.rs

```diff
@@ -244,6 +257,18 @@ pub fn install_binaries(
     }
     // Copy binaries
     for binary in binaries {
+        let missing_feature = binary
+            .required_features
+            .iter()
+            .find(|feature| !features.contains(*feature))
+            .is_some();
```

Comment:
Nit, I think `clippy` would say that `find([...]).is_some()` can be changed into `any([...])`.